task/zap-baseline/README.md

```diff
@@ -0,0 +1,3 @@
+# Zap proxy
+
+[Zap proxy](https://github.com/zaproxy/zaproxy) baseline is a command line tool for security scanning endpoints.
```

Comment:
Couldn't come up with a super descriptive what zap does.
The tools is created by the OWASP community and it performance security scanning on endpoints ^^

Trying to explain how it does it feels a bit overkill to me and to be honest I don't know, I'm just a happy user of the tool. But if you feel the need I can try to understand deeper how the tool actually works and try to describe it.

Reply:
What I am trying to say over here is that this extra README in `task/zap-baseline` is not required since we have the README inside `task/zap-baseline/0.1`

Reply:
According to the TEP, there can be a generic readme for all the versions, but that can be optional.

Can you please see this [example](https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md#example-of-a-resource-from-catalog) once and the comments added to the fields in the yaml file